CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## XAVIER development version
 
+## XAVIER 3.1.5
+
+- Updated base docker image for sequenza (#130, @dnousome)
```

Comment:
Hey @dnousome @samarth8392, we normally link the PR number in the changelog rather than the issue number, so this one would have been 140 rather than 130. If you guys could start following that format, that'd be great so we can have consistency across our pipelines. Thanks!